mlperf_logging/result_summarizer/result_summarizer.py

```diff
@@ -272,6 +272,21 @@ def _assert_in_desc_and_return(desc, desc_keys, query=None):
     return desc[desc_keys[0]] if query is None else query(desc)
 
 
+def _get_scaling_factor(folder):
+    """Apply scaling factor if benchmark dir has json file scaling.json.
+
+    This file can be added only during the review period if the review
+    committee decides score should be scaled.
+    """
+    scaling_file = os.path.join(folder, 'scaling.json')
+    if os.path.exists(scaling_file):
```

Comment:
1. `os.path.exists` would return `True` if the path points to a directory.
2. Below is a little cleaner: 
```python
    scaling_factor = 1.0
    scaling_file = os.path.join(folder, 'scaling.json')
    if os.path.isfile(scaling_file):
        contents = _read_json_file(scaling_file)
        scaling_factor = contents['scaling_factor']
        print(f'NOTICE: Applying scaling factor {scaling_factor} to dir {folder}')
    return scaling_factor
```